setup.py

```diff
@@ -43,7 +42,7 @@ def read(fname):
     },
     version=version,
     description="Tools for tuning optical parametric amplifiers and multidimensional spectrometers.",
-    long_description=read("README.rst"),
+    long_description=read("README.md"),
```

Comment:
```suggestion
    long_description=read("README.md"),
    long_description_content_type="text/markdown",
```

This tells the pypi website to treat the contents as markdown

Reply:
Is it equivalent to put it in setup.cfg?  Wondering if I should take my changes out of there...

Reply:
I think it is, I missed that change... theres a move away from setup.cfg in the python packaging ecosystem in general, but like I don't expect you to take it out here, so we can assume it is fine until proven otherwise.